Updated Container-Runtime.md (#30169)
kubernetes/website: content/en/docs/setup/production-environment/container-runtimes.md

- [CRI-O](#cri-o)
- [Docker](#docker)

> Dockershim has been [deprecated](https://kubernetes.io/blog/2020/12/02/dockershim-faq/)
> as of [version 1.20.0](https://github.com/kubernetes/kubernetes/pull/94624)
> and is likely to be dropped of
> by 1.24.0, therefore to use Docker you need to be running an older version of
> Kubernetes. You can check out this [documentation](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/)
> to understand how this deprecation might affect you. For migrating from dockershim you can follow
> [this](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/_print/).
> A robust guide on migrating from Dockershim to other CRIs will be released soon.
> 

Reviewer: ```suggestion
Dockershim has been [deprecated](https://kubernetes.io/blog/2020/12/02/dockershim-faq/)
as of [version 1.20.0](https://github.com/kubernetes/kubernetes/pull/94624)
and is likely to be dropped of by 1.24.0, therefore to use Docker you need to be running an older version of
Kubernetes. 
To know how this deprecation affects you and how to migrate from dockershim you can follow [this document](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/_print/).
```

Reviewer: Also its better if we add this in `notes` section

Author: @adisky, should I create another note section for this or add it in the existing one?

- [CRI-O](#cri-o)
- [Docker](#docker)

{{< note >}}

> Docker container runtime (Dockershim) has been [deprecated](
> /blog/2020/12/02/dockershim-faq/)
> as of [version 1.20](
> /blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)
> and is planned to be removed in by 1.24, therefore to use Docker you need to
> be running an older version of Kubernetes.
> Read the [Migrating from dockershim](
> /docs/tasks/administer-cluster/migrating-from-dockershim/) section to learn
> more about migrating to an alternative container runtime.

Reviewer 2: ```suggestion
Kubernetes has a special integration with Docker, `dockershim`. This special integration
was formally [deprecated](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)
in the v1.20 release of Kubernetes, and is scheduled to be removed in the v1.24 release.

For more information, read
[Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/), or
visit [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/).
```

- [CRI-O](#cri-o)
- [Docker](#docker)

{{< note >}}

> Docker container runtime (Dockershim) has been [deprecated](

Reviewer 2: > for links, we'd prefer keeping them on a single line.

This is still true. Put the line break after the `)` that ends the hyperlink, not after the `(`.

https://kubernetes.io/blog/2020/12/02/dockershim-faq/)
as of [version 1.20.0](https://github.com/kubernetes/kubernetes/pull/94624)

> and is likely to be dropped ofby 1.24.0, therefore to use Docker you need to be

Reviewer 3: ```suggestion
and is likely to be dropped off by 1.24.0, therefore to use Docker you need to be
```

and is planned to be removed in by 1.24, therefore to use Docker you need to
be running an older version of Kubernetes.
You can check out this [documentation](
https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/)
to understand how this deprecation might affect you. For migrating from
dockershim you can follow [this](
https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/) 

> document to migrate from dockershim.

Author: @sftim I understood what you are saying.  And about the summary text, should I add it in a seperate PR for more clarity or adding it in this PR itself is fine?

{{< tabs name="tab-cri-containerd-installation" >}}
{{% tab name="Linux" %}}

> 1. Install the `containerd.io` package from the official Docker repositories.

Reviewer: why we are changing these lines?
